src/cli/ssh-key.ts

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env node
+
+import * as fs from 'fs';
+import * as path from 'path';
+import axios from 'axios';
+
+const API_BASE_URL = process.env.GIT_PROXY_API_URL || 'http://localhost:3000';
+const GIT_PROXY_COOKIE_FILE = path.join(
+  process.env.HOME || process.env.USERPROFILE || '',
+  '.git-proxy-cookies.json',
+);
+
+interface ApiErrorResponse {
+  error: string;
+}
+
+interface ErrorWithResponse {
+  response?: {
+    data: ApiErrorResponse;
+    status: number;
+  };
+  code?: string;
+  message: string;
+}
+
+async function addSSHKey(username: string, keyPath: string): Promise<void> {
```

Comment:
Just noticed that there's duplicate logic for `addSSHKey` and `removeSSHKey` - the same API calls are happening in `git-proxy-cli/index.ts`. We should figure out if we want to keep the standalone SSH key manager script or have it all included in the CLI.